### DISCUSSION TITLE
Must have features

### DISCUSSION THREAD

**Discussion by kleozzy** [pH7Software/pH7-Social-Dating-CMS#957] [Ideas/Suggestions]
Hey man!

First of all amazing work for doing this. It's already great. Here are some features I have seen that really needed for a dating website to be effective:

1. Ability to limit messages/time for various user types, so instead of blocking IM you can limit it to 10/day and so on.
2. Friends and photo access. Have visibility status for media (public, friends only, and private)
3. Upload a picture for verification (with a gesture or holding paper) and assign a verification badge ( very useful for dating websites)
4. Filter words and numbers, emails in chats ( to prevent offsite comms)
5. Ability to add custom genders
6. Ability to Pay via Paysafe cards
7. Couple profiles functionality

**Comment by mateable** [comment]
What about users can customize there profiles
